### DISCUSSION TITLE
Secret Scanning automatic validity check on private repositories

### DISCUSSION THREAD

**Discussion by VollRagm** [community/community#66612] [Code Security] [answered]
### Select Topic Area

Question

### Body

Hi,

if I enable the "Automatically verify if a secret is valid by sending it to the relevant partner" in the organization settings, are the partners going to revoke the secrets, even though the repositories are private?

Based on the documentation partners only revoke secrets found in public repositories, but I would like to make sure.

Thanks.

**Comment by psehgal** [comment, marked as answer]
@VollRagm, partners shouldn't be revoking secrets when we test their validity.
